sample_size/multiple_testing.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+
+import numpy as np
+from statsmodels.stats.multitest import multipletests
+
+from sample_size.metrics import BaseMetric
+
+
+class MultipleTestingMixin:
+    """
+    This class calculates sample size required under the case of multiple testing
+
+    Attributes:
+    metrics: a list of BaseMetric registered by users
+    variants: number of variants, including control
+    alpha: statistical significance
+    power: statistical power
+
+    """
+
+    metrics: List[BaseMetric]
+    alpha: float
+    power: float
+    variants: int
+
+    def get_multiple_sample_size(self, lower: float, upper: float, depth: int = 0) -> int:
+        MAX_RECURSION_DEPTH = 20
+        EPSILON = 0.025
```

Comment:
Could you make a JIRA ticket for this?

```suggestion
        EPSILON = 0.025 #  TODO(any): make this configurable by the user ML-XXXX
```